src/components/GameClock.cpp

```diff
@@ -30,80 +34,95 @@ namespace REGoth
   bs::INT32 GameClock::getHour() const
   {
     float elapsedSecondsOfDay = mElapsedIngameSeconds - (getDay() * SECONDS_IN_A_DAY);
-    
+
     return bs::Math::floorToPosInt(elapsedSecondsOfDay / SECONDS_IN_AN_HOUR);
   }
 
   bs::INT32 GameClock::getMinute() const
   {
-    float elapsedSecondsOfHour = mElapsedIngameSeconds - (getDay() * SECONDS_IN_A_DAY) - (getHour() * SECONDS_IN_AN_HOUR);
+    float elapsedSecondsOfHour =
+        mElapsedIngameSeconds - (getDay() * SECONDS_IN_A_DAY) - (getHour() * SECONDS_IN_AN_HOUR);
 
     return bs::Math::floorToPosInt(elapsedSecondsOfHour / SECONDS_IN_A_MINUTE);
   }
 
+  float GameClock::getDayRatio() const
+  {
+    float elapsedSecondsOfDay = mElapsedIngameSeconds - (getDay() * SECONDS_IN_A_DAY);
```

Comment:
Maybe create a function `getCurrentTimeOfDayInSeconds` or so as the exact same snippet is used in `GameClock::isTime`.

Reply:
Done!